tshirts.c

```diff
@@ -17,6 +17,13 @@ int main() {
     assert(size(37) == 'S');
     assert(size(40) == 'M');
     assert(size(43) == 'L');
+    
+    //newly added tests
+    assert(size(38) == 'S'); //Test to check the boundary value 38 for "s" size
+    assert(size(38) == 'M'); //Test to check the boundary value 38 for "m" size

```

Comment:
Do you expect size 38 to be both S and M , and similarly for size 42?

Reply:
@prithamvshetty , either 38 can be S or M. similarly 42 can be M or L
updated the test cases as per this understanding from your review comments.

Reply:
If the production code returns M and S, then it is non deterministic.
You make an assumption and provide an assert for same. You have to be clear about the expectation from production code. For ex, size >=38 is M, and hence I will have 2 assets, size(38) == M and one above value.

Reply:
Let me know if you still have doubts 👍

Reply:
@prithamvshetty , understood. Modified the test cases accordingly.

Reply:
👍